### PR TITLE
cpu and memory variables introduced

### PR DESCRIPTION
**Pull request by yauheni-sem**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -144,4 +144,16 @@ variable "listener_settings" {
...
     listener_arn = ""
   }
-}
\ No newline at end of file
+}
+
+variable "cpu" {
+  description = "Number of cpu units used by the task"
+  type        = number
+  default     = 0
```
Comment (YAndrew91) [inline variables.tf:152]: @adys @yauheni-sem i am not sure if zero value is handled in the same way as no value

Comment (yauheni-sem) [reply variables.tf:152]: Hm, the 0 value should be equivalent to unset: https://github.com/hashicorp/terraform-provider-aws/blob/834839aec8a8b62896bcd291a3cee547885f2993/internal/service/ecs/task_definition_equivalency.go#L64

**Review by YAndrew91 — APPROVED** [review]

File: variables.tf (hunk 1 above)
Comment (adys) [reply variables.tf:152]: @yauheni-sem you shared a reference to the part where container definition is set. [Here is the cpu set](https://github.com/hashicorp/terraform-provider-aws/blob/834839aec8a8b62896bcd291a3cee547885f2993/internal/service/ecs/task_definition.go#L84) on the Task level. 

Additionally, I've tested it. (btw, feel free to use [dummy-service](https://github.com/craft-machine/dummy-service) for such tests). Just run terraform locally and play around.

Long story short, let's set the default to **null** instead of 0.

**Review by adys — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 1 above)
Comment (yauheni-sem) [reply variables.tf]: Thanks for the check and suggestion! I've updated the default values to `null`

**Review by adys — APPROVED** [review]
